[Blog] week 27 (#141)
packit-service/packit.dev: content/posts/July-2020.md

---
title: "July 2020"
date: 2020-07-03
draft: false
---

## Week 27 (June 29th - July 3rd)

### packit

> * We introduced a new command `packit validate-config` which validates the contect of a package config

Reviewer: ```suggestion
* We introduced a new command `packit validate-config` to validate the content of a package configuration files
```

Reviewer 2: I'd even say 

    `packit.yaml` in your project

Reviewer 3: But we support also different names:

```python
CONFIG_FILE_NAMES = [
    ".packit.yaml",
    ".packit.yml",
    ".packit.json",
    "packit.yaml",
    "packit.yml",
    "packit.json",
]
```

  (contributed by [@TomasJani](https://github.com/TomasJani), [#826](https://github.com/packit-service/packit/pull/826)).

### packit-as-a-service

* You are now able to override configuration for builds, releases and other jobs (by [@TomasTomecek](https://github.com/TomasTomecek), [#676](https://github.com/packit-service/packit-service/pull/676)).

>  Read more about this feature [here](/docs/configuration/#overriding-global-parameters).

Reviewer 2: well done! linking to docs themselves
